site/scripts/sync-algolia-index.ts

```diff
@@ -9,7 +9,7 @@ import siteMap from "../site-map.json" assert { type: "json" };
 
 const monorepoRoot = path.resolve(
   path.dirname(fileURLToPath(import.meta.url)),
-  "..",
+  "../..",
```

Comment:
We moved this file from `scripts` to `site/scripts` in https://github.com/blockprotocol/blockprotocol/pull/375, but I did not spot this new issue with `monorepoRoot`. cc @CiaranMn